WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/ProductFormRemoteActionUseCaseTests.swift

```diff
@@ -100,12 +106,81 @@ final class ProductFormRemoteActionUseCaseTests: XCTestCase {
     }
 
     // MARK: - Editing a product (`addProduct`)
+    func test_editing_product_and_password_when_WooCommerce_version_is_below_8_1_returns_false() {
+        // Given
+        let plugin = SystemPlugin.fake().copy(siteID: siteID,
+                                              name: "WooCommerce",
+                                              version: "8.0.0",
+                                              active: true)
+        storageManager.insertSampleSystemPlugin(readOnlySystemPlugin: plugin)
+
+        let sut = ProductPasswordEligibilityUseCase(stores: storesManager, storageManager: storageManager)
```

Comment:
I'm confused - this test should be added in a separate test file for `ProductPasswordEligibilityUseCase` instead of `ProductFormRemoteActionUseCase`.

Reply:
Updated here 31561985e4a6fa81f0fc08a879163acf13bb00b6 and here 8932bdff00a33fdafcce171b2443e69b2c08b571